CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ This is the log of notable changes to the Expo client that are developer-facing.
 
 ### 🐛 Bug fixes
 
+- Fixed `uuid`'s deprecation of deep requiring ([#8114](https://github.com/expo/expo/pull/8114) by [@actuallymentor](https://github.com/actuallymentor) )
```

Comment:
```suggestion
- Fixed `uuid`'s deprecation of deep requiring ([#8114](https://github.com/expo/expo/pull/8114) by [@actuallymentor](https://github.com/actuallymentor))
```
Move this change to `packages/expo-constants/CHANGELOG` as this affects `expo-constants` module only and not whole repository 😉